NuGet.Samples.Interop/NuGet.Samples.Interop.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>NuGet.Samples.Interop</PackageId>
+    <TargetFramework>net46</TargetFramework>
+    <Shipping>true</Shipping>
+    <PackProject>true</PackProject>
+    <IncludeInVsix>true</IncludeInVsix>
```

Comment:
Shipping and IncludeInVsix are are our internal properties for NuGet.Client. I'm not sure if PackProject is or not, but this doesn't need to auto pack for the user, they can just run dotnet pack on it. So all 3 of these should be removed to avoid confusion.